webpack.config.js

```diff
@@ -0,0 +1,78 @@
+const webpack = require("webpack");
+const path = require("path");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+const package = require("./package");
+const widgetName = package.widgetName;
+
+const widgetConfig = {
+    entry: `./src/components/${widgetName}Container.ts`,
+    output: {
+        path: path.resolve(__dirname, "dist/tmp"),
+        filename: `src/com/mendix/widget/custom/dropdownsort/${widgetName}.js`,
```

Comment:
Don't use the widget name in the webpack config.
``` ts
const packageName = package.widgetName.toLowerCase();
```

Reply:
Done

Reply:
Changing the to lowercase works on local host but does not work on the sandbox. so it is not a good idea to do so